packages/SwingSet/test/test-vpid-kernel.js

```diff
@@ -3,7 +3,7 @@ import { test } from '../tools/prepare-test-env-ava.js';
 
 import anylogger from 'anylogger';
 import { assert, details as X } from '@agoric/assert';
-import { WeakRef, FinalizationRegistry } from '../src/weakref.js';
+/* global WeakRef, FinalizationRegistry */
```

Comment:
does the `global` comment need to be on the first line of the file?

Reply:
Not to pass lint but I'd say yes for style. I meant to move these after find-replace but missed this one.